src/sass/time-grid.scss

```diff
@@ -135,7 +135,19 @@
   z-index: 3;
   left: 0;
   right: 0;
-  height: 1px;
+  height: 2px;
```

Comment:
are these changes missing in the `.less` files?

Reply:
(hurried typing noises) idk what ur talkin about....